R/Class-SAMS.R

```diff
@@ -34,15 +34,82 @@
 #
 
 #
-# 
+# SamsShape ----
 #
 setClass("SamsShape",
-         representation(point = "GmlPoint"),
-         prototype = list(point = NULL),
-         contains = "GmlGeometry",
-         validity = function(object) {
-           #print("Entering validation: SamsShape")
-           return(TRUE)
-         }
+         slots = c(point = "GmlPoint"),
+         # TODO is this class inheritence required? Fields of GmlGeometry are contained in GmlPoint.
+         contains = "GmlGeometry"
 )
 
+SamsShape <- function(point) {
+  new("SamsShape", point = point)
+}
+
+.toString.SamsShape <- function(x, ...) {
```

Comment:
I see the point that this is more useful during development. I'm trying hard to remember why I put all in one file when I wrote the package, probably because they involved a lot of copy-pasting at the time.

Let's keep these where they are then!